woocommerce/wc-admin
# Fix and refactor explat polling to use setTimeout

This PR refactors anon polling in explat package to fix uninitialized const variable access improve execution flow in its polling for cookie. Previously, there were couple of issues identified:

1. The `const` variable was accessed when the first poll immediately receives cookie, thus `setInterval` will attempt to access an interval handle without it ever being initialized.
2. When the first poll is able to resolve the promise, it does not clear the interval (since no handle exists) thus could potentially cause a temporary memory leak. This was the case when our previous transpiler changed `const anonIdPollingInterval` to `var anonIdPollingInterval`, sneaking in this bug under a trojan horse.

`setTimeout` is also better for our polling since it would never have 2 simultaneous instances of a poll run, which could theoretically happen with `setInterval` where a future poll run is not tied to the execution flow of the existing run.

### Detailed test instructions:

1. Go to WooCommerce > Home
1. Make sure no error in console
